lib/apytypes/_apytypes.pyi

```diff
@@ -808,7 +808,133 @@ class APyFixedArray:
         """
 
     def squeeze(self, axis: int | tuple | None = None) -> APyFixedArray:
-        """Squeeze the matrix, removing dimensions of size 1."""
+        """
+        Removes dimensions of size 1 at the specified axis/axes, if no axís is given removes all dimensions with size 1.
```

Comment:
### [numpy.squeeze](https://numpy.org/doc/stable/reference/generated/numpy.squeeze.html)
> Remove axes of length one from a.

Reply:
I think one can split this into two sentences. (And have the second sentence as an extended description, one empty line down.)